### PR TITLE
Add our weights for raft_small on C+T

### PR DESCRIPTION
**Pull request by NicolasHug**
This PR adds our own pre-trained weights for Raft-small. They are a bit better on the clean pass of sintel, about the same on the final pass, and a tiny bit better on kitti.

I also updated the META info of the raft-large models regarding kitti, which I forgot to do before.

with ` --pretrained`

```
(raft) ➜  vision git:(raft_small_weights_ours) ✗ torchrun --nproc_per_node 8 --nnodes 1 references/optical_flow/train.py --val-dataset kitti --batch-size 10 --model raft_small --dataset-root ../downloads --pretrained
Kitti val epe: 8.0209	1px: 0.5074	3px: 0.7333	5px: 0.7963	per_image_epe: 7.5978	f1: 25.2369
```
with `--weights Raft_Small_Weights.C_T_V2`
```
(raft) ➜  vision git:(raft_small_weights_ours) ✗ torchrun --nproc_per_node 8 --nnodes 1 references/optical_flow/train.py --val-dataset kitti --batch-size 10 --model raft_small --dataset-root ../downloads --weights Raft_Small_Weights.C_T_V2

Kitti val epe: 8.0209	1px: 0.5074	3px: 0.7333	5px: 0.7963	per_image_epe: 7.5978	f1: 25.2369
```

cc @datumbox

### CODE REVIEW

File: torchvision/prototype/models/optical_flow/raft.py [hunk 1]
```diff
@@ -46,6 +48,8 @@ class Raft_Large_Weights(WeightsEnum):
             "recipe": "https://github.com/pytorch/vision/tree/main/references/optical_flow",
             "sintel_train_cleanpass_epe": 1.3822,
             "sintel_train_finalpass_epe": 2.7161,
+            "kitti_train_per_image_epe": 4.5118,
+            "kitti_train_f1-all": 16.0679,
```
Comment (datumbox) [inline torchvision/prototype/models/optical_flow/raft.py:52]: Out of curiosity is this the standard F1 score?

File: torchvision/prototype/models/optical_flow/raft.py [hunk 2]
```diff
@@ -87,10 +91,25 @@ class Raft_Small_Weights(WeightsEnum):
             "recipe": "https://github.com/princeton-vl/RAFT",
             "sintel_train_cleanpass_epe": 2.1231,
             "sintel_train_finalpass_epe": 3.2790,
+            "kitti_train_per_image_epe": 7.6557,
+            "kitti_train_f1-all": 25.2801,
+        },
+    )
+    C_T_V2 = Weights(
+        # Chairs + Things
+        url="https://download.pytorch.org/models/raft_small_C_T_V2-01064c6d.pth",
```
Comment (datumbox) [inline torchvision/prototype/models/optical_flow/raft.py:100]: Typo?
```suggestion
        url="https://github.com/pytorch/vision/tree/main/references/optical_flow",
```

**Review by datumbox — APPROVED** [review]
Thanks @NicolasHug. I left one comment but stamping to unblock.

File: torchvision/prototype/models/optical_flow/raft.py (hunk 1 above)
Comment (NicolasHug) [reply torchvision/prototype/models/optical_flow/raft.py:52]: Not really, it's a kitti-specific metric, I saw it in different papers (and it's what they report in the leaderboards)

```py
"f1": ((epe > 3) & (relative_epe > 0.05)).float().mean().item() * 100,
```

File: torchvision/prototype/models/optical_flow/raft.py (hunk 2 above)
Comment (datumbox) [reply torchvision/prototype/models/optical_flow/raft.py]: Stupid me, I commented on the wrong line. The remark was for the "recipe" URL which is couple of lines below. I spoke with Nicolas offline and we will fix this on a follow up PR.
